etc/docker/kernel-scala/Dockerfile

```diff
@@ -1,5 +1,15 @@
 FROM elyra/spark:v2.4.1
 
+# Depending on the version of Kubernetes, some Spark jobs get
+# failures attempting to create executors. These steps update
+# the kubernetes client to 4.4.2 and will not be necessary
+# once Spark 2.4.5 or 3.0 have been released.
+# See https://issues.apache.org/jira/browse/SPARK-28921
+RUN rm -f $SPARK_HOME/jars/kubernetes-*.jar
+ADD https://repo1.maven.org/maven2/io/fabric8/kubernetes-client/4.4.2/kubernetes-client-4.4.2.jar $SPARK_HOME/jars
+ADD https://repo1.maven.org/maven2/io/fabric8/kubernetes-model/4.4.2/kubernetes-model-4.4.2.jar $SPARK_HOME/jars
+ADD https://repo1.maven.org/maven2/io/fabric8/kubernetes-model-common/4.4.2/kubernetes-model-common-4.4.2.jar $SPARK_HOME/jars
+
```

Comment:
Will this replace or duplicate these jars on the spark deployment? if there are multiple versions the used one will be the first loaded which differes in different OS.

Reply:
It replaces the files. The `RUN` command (line 8) removes the existing versions.